src/registry.jl

```diff
@@ -13,8 +13,9 @@ function get_registry(basepath; registry=DOCS_REGISTRY, sync = true)
     if sync
         try
             rm(joinpath(basepath, "DocumentationGeneratorRegistry"), force = true, recursive = true)
-            cd(basepath)
-            run(`git clone --depth=1 $(registry) DocumentationGeneratorRegistry`)
+            cd(basepath) do
+                run(`git clone --depth=1 $(registry) DocumentationGeneratorRegistry`)
+            end
```

Comment:
```suggestion
            run(`git clone --depth=1 $(registry) $(joinpath(basepath, "DocumentationGeneratorRegistry"))`)
```
is probably even nicer.